infra/sync/src/main/java/com/simprints/infra/sync/usecase/CleanupDeprecatedWorkersUseCase.kt

```diff
@@ -37,12 +37,10 @@ internal class CleanupDeprecatedWorkersUseCase @Inject constructor(
 
     private fun tagsForDeprecatedWorkers() = listOf(
         "TAG_PEOPLE_SYNC_WORKER_TYPE_DOWN_COUNTER", // 2023.1.0
-        "TAG_PEOPLE_SYNC_WORKER_TYPE_DOWNLOADER", // 2023.1.0
         "TAG_PEOPLE_SYNC_WORKER_TYPE_END_SYNC_REPORTER", // 2023.1.0
```

Comment:
I think reporter workers also should not be cancelled.

Reply:
Done